yarn.lock

```diff
@@ -2670,11 +2670,16 @@ gopd@^1.0.1, gopd@^1.2.0:
   resolved "https://registry.yarnpkg.com/gopd/-/gopd-1.2.0.tgz#89f56b8217bdbc8802bd299df6d7f1081d7e51a1"
   integrity sha512-ZUKRh6/kUFoAiTAtTYPZJ3hw9wNxx+BIBOijnlG9PnrJsCcSjs1wyyD6vJpaYtgnzDrKYRSqf3OO6Rfa93xsRg==
 
-govuk-frontend@^5.0.0, govuk-frontend@^5.4.0:
+govuk-frontend@^5.0.0:
```

Comment:
should we not use the most recent V.5.4 ? 
... anyway, we can update in a different PR

Reply:
Good spot, MoJ-frontend was requesting "caret 5.0.0" and we were resolving that to  5.4.0. I've updated it so that now when MoJ frontend requests it, we resolve to 5.10.2, which is the latest.